chapter03/n27.py

```diff
@@ -0,0 +1,91 @@
+"""
+26の処理に加えて，テンプレートの値からMediaWikiの内部リンクマークアップを除去し，テキストに変換せよ
+"""
+import re
+
+
+def remove_markup(target):
+    '''マークアップの除去
+    強調マークアップを除去する
+
+    引数：
+    target -- 対象の文字列
+    戻り値：
+    強調マークアップを除去した文字列
+    '''
+
+    # 強調マークアップの除去
+    markup_pattern = re.compile(r'''
+        (\'{2,5})  # 2~5個の'（マークアップの開始）
+        (.*?)  # 任意の1文字以上（対象の文字列）
+        (\1)  # 1番目のキャプチャと同じ（マークアップの終了）
+        ''', re.MULTILINE + re.VERBOSE)
+    target = markup_pattern.sub(r'\2', target)
```

Comment:
一時変数はわざわざ変数に入れずそのままreturnしちゃってくれ
```suggestion
     return markup_pattern.sub(r'\2', target)
```

説明用の中間変数として使うケースもあるが、だいたいの一時変数はインライン化してくれ